test/helpers/data/staging-workload-helper.js

```diff
@@ -111,24 +114,45 @@ module.exports.deleteAll = function () {
     })
 }
 
-function mapForInsert (record) {
+function mapForInsert (record, isT2A = false) {
   var row = {}
   for (let key in record) {
-    if (key === 'communityTiers') {
+    if (key === 'communityTiers' && !isT2A) {
```

Comment:
I don't see how the mapping process is different for t2a cases?

Reply:
the mapping is based on the key. t2a cases will contain case summary of t2aCommunityTiers, etc. The underline mapping is the same. We need this to ensure we only insert either t2a cases or non-t2a cases.